src/client/views/components/todoItem.jsx

```diff
@@ -19,6 +19,14 @@ export default React.createClass({
         if (this.state.inEdit) {
             this.focusOnHeading();
         }
+
+        let el = this.getDOMNode();
+        let isVisible = (el.getBoundingClientRect().top >= 0) && (el.getBoundingClientRect().bottom <= window.innerHeight);
+        if (!isVisible) {
+            setTimeout(() => {
+                window.scrollTo(0,document.body.scrollHeight);
+            }, 100);
```

Comment:
@mrmckeb http://stackoverflow.com/questions/26556436/react-after-render-code/34999925 
I'm not sure if this is an animation issue. How would I know how long it takes for the height to fully animate though?